src/platform/packages/shared/presentation/presentation_containers/interfaces/presentation_container.ts

```diff
@@ -73,6 +74,14 @@ export const apiIsPresentationContainer = (api: unknown | null): api is Presenta
   );
 };
 
+export interface HasSections extends CanAddNewSection {
+  getPanelSection$: (uuid: string) => Observable<string | undefined>;
+}
+
+export const apiHasSections = (api: unknown): api is HasSections => {
+  return typeof (api as HasSections)?.getPanelSection$ === 'function';
+};
```

Comment:
We use `getPanelSection$` so that control panel knows when it has changed sections and can update its `group` filter meta data accordingly.